docs/reference/settings/security-settings.asciidoc

```diff
@@ -1633,6 +1633,23 @@ at the OpenID Connect Provider.
 The OAuth 2.0 Client Secret that was assigned to {es} during registration
 at the OpenID Connect Provider.
 
+// tag::rp-client-auth-method-tag[]
+`rp.client_auth_method` {ess-icon}::
+(<<static-cluster-setting, Static>>)
+The Client authentication method used by elasticsearch as a client to authenticate
```

Comment:
Feels one of the two occurrences of "client" is redundant:
```suggestion
The authentication method used by {es} as a client to authenticate
```

Reply:
I'll change the {es} thanks ! but it's called "Client authentication" in OpenID Connect , so I will leave this here